migrator/src/main/resources/templates/dataset.json.ftl

```diff
@@ -57,31 +57,42 @@
   "nodes" : [
   <#list dataset.attachments as attachment>
   {
-    "name":"publicationsystem:Files-v2",
-    "primaryType":"publicationsystem:resource",
+    "name":"publicationsystem:Files-v3",
+    "primaryType" : "publicationsystem:attachment",
     "mixinTypes":[],
-    "properties":[ {
-        "name":"hippo:filename",
-        "type":"STRING",
-        "multiple":false,
-        "values":[ "${attachment.title}" ]
-      }, {
-        "name":"jcr:data",
-        "type":"BINARY",
-        "multiple":false,
-        "values":[ "file://${attachment.filePath}" ]
-      }, {
-        "name":"jcr:mimeType",
-        "type":"STRING",
-        "multiple":false,
-        "values":[ "${attachment.mimeType}" ]
-      }, {
-        "name":"jcr:lastModified",
-        "type":"DATE",
-        "multiple":false,
-        "values":[ "0001-01-01T12:00:00.000Z" ]
-      } ],
-    "nodes":[]
+    "properties" : [ {
```

Comment:
Cosmetic: "propeties" array could use some indentation - as is, at first glance I thought that we have the "name" property twice at the top object level :-)

Reply:
👍